addons/stock/models/stock_picking.py

```diff
@@ -680,6 +680,11 @@ def action_assign(self):
         package_level_done.write({'is_done': False})
         moves._action_assign()
         package_level_done.write({'is_done': True})
+
+        # run scheduler for moves are not fully reserved
+        for move in moves.filtered(lambda m: m.state in ['confirmed', 'partially_available']):
+            move.product_id.orderpoint_ids.filtered(lambda op: op.location_id == move.location_id and op.trigger == 'auto').action_replenish(raise_user_error=False)
```

Comment:
you can use _procure_orderpoint_confirm instead of action_replenish
you should use op.location_id parent_of move.location_id

Reply:
change applied